jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionIdManager.java

```diff
@@ -489,7 +525,7 @@ public boolean idInUse(String sessionId)
         /*
          * optimize this query to only return the valid variable
          */
-        DBObject o = _sessions.findOne(new BasicDBObject("id",sessionId), __valid_true);
+        DBObject o = _sessions.findOne(new BasicDBObject("id",sessionId));
```

Comment:
I can back that out. Accidental removal when re-orging after adding the indexes. Oddly that doesn't seem to overly impact perf.